_sass/_templates/home.scss

```diff
@@ -48,15 +48,15 @@ $tablet-md: 760px;
     align-self: flex-start;
     margin-top: 0;
     max-width: $hero-graphic-md;
-    order: 4;
+    order:4;
```

Comment:
To follow a consistent style there should be a space after the colon on each of these.